Simplenote.xcodeproj/project.pbxproj

```diff
@@ -84,7 +84,7 @@
 		37D4DD6A20B3574D00C225EA /* WPAuthHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = 37D4DD6820B3574C00C225EA /* WPAuthHandler.m */; };
 		37F742EB202A382400A47D3A /* AboutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 37F742EA202A382400A47D3A /* AboutViewController.swift */; };
 		37F742EC202A382400A47D3A /* AboutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 37F742EA202A382400A47D3A /* AboutViewController.swift */; };
-		3FAC7B25254BBCA600B47276 /* SimplenoteInterlinks in Frameworks */ = {isa = PBXBuildFile; productRef = 3FAC7B24254BBCA600B47276 /* SimplenoteInterlinks */; };
+		3FAC7B25254BBCA600B47276 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; productRef = 3FAC7B24254BBCA600B47276 /* SwiftPackageProductDependency */; };
```

Comment:
🤔  A mention of a generic "BuildFile" name here is usually the sign of a corruption in the pbxproj (typically a file that was referenced by a UUID in this Build Phase but the file and its UUID were removed from the Objects List at the top of the pbxproj but the UUID pointing to it wasn't – probably the result of conflict resolution gone bad).

In such cases, there is usually a "BuildFile" entry with a generic document icon when you open the corresponding build phase in Xcode's UI, or even sometimes there's just an empty line (that mostly happen when the entry is `(null)` in the pbxproj tho, not `BuildFile`)